salt/grains/core.py

```diff
@@ -1927,7 +1927,7 @@ def fqdns():
             fqdns.add(socket.gethostbyaddr(ip)[0])
         except (socket.error, socket.herror,
             socket.gaierror, socket.timeout) as e:
-            log.error("Exception during resolving address: " + str(e))
+            log.info("Exception during resolving address: " + str(e))
```

Comment:
@rallytime but this seems to me a wrong fix:

- It sets `.info` after the exception.
- It uses concatenation instead of formatting through the Unicode

Proper fix would be to move this to a debug level and use proper formatting:

```python
except (socket.error, socket.herror, socket.gaierror, socket.timeout) as err:
    log.debug('Exception during resolving address: %s', err)
except Exception as err:
    log.exception(err)
```